### ISSUE TITLE
LTC intake approval script--Lookback period is based on incorrect data

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#483]
HCPM  19.40.15 - Lookback Period
Determining the MA Lookback Period
 The lookback period is a specific period of time prior to a baseline date. A new lookback period is established each time a person requests MA payment of LTC services following a break in long-term care (LTC) services. The baseline date is the date in which both of the following conditions are met: 
1. a person is residing in an LTCF or, for a person requesting services through a home and community-based waiver program, the date a screening occurred that indicated a need for services provided through a home and community-based services waiver program AND
2.the person’s initial request month for MA payment of LTC services.

So long story short, the current script is pulling the 'lookback period' from the date of application.  Because of complex factors, we will need the workers to enter which date is the "baseline date", and then the script can determine the lookback period based on this date.  
Baseline date: 03/01/15 (based on date of application 03/18/2015, and client meeting LOC)
Lookback period: 03/01/10-02/28/15

I can take this one
